### DISCUSSION TITLE
Artist Covers, the missing piece

### DISCUSSION THREAD

**Discussion by Node815** [blackcandy-org/blackcandy#402] [General]
Based on this request 4 years ago, it may have been put on the very far back burner or forgotten.  I installed this tonight and am super impressed at how fast it indexed my 6.7k song library almost instantly, so this was a big plus for me until I saw the missing artist covers.   

The request is here:
https://github.com/blackcandy-org/blackcandy/issues/63

Has there been any process made toward this?  

I did set up a discogs account to see if it would bring in the images, but so far no luck with that either.

**Comment by volcs0** [comment]
Bumping this up - amazed at performance but would love to see artist covers - they are all blank. I did add my discogs API key. Any updates?

**Reply by volcs0** [reply]
Actually, you can manually add the artist cover - so it would great to do this automatically.
